README.md

```diff
@@ -45,152 +21,53 @@ though we include a summary of them here:
 * Binary fixed-point math
 * Restrictions on reentrancy
 * Static looping
+* Module imports
+* Usage of [YUL](https://docs.soliditylang.org/en/latest/yul.html) IR to target both EVM and eWASM
+* WASM compiler binaries for enhanced portability and in-browser compilation of
+  Fe contracts
+* Implementation in a powerful, systems-oriented language (Rust) with strong safety guarantees to reduce risk of compiler bugs
+
+Additional information about design goals and background can be found in the [official announcement](https://snakecharmers.ethereum.org/fe-a-new-language-for-the-ethereum-ecosystem/).
+
```

Comment:
A bit of backstory is conserved in the announcement post so I think its enough to link to it here